### DISCUSSION TITLE
utf-8 greek problem java(netbeans)

### DISCUSSION THREAD

**Discussion by Chrisdamia99** [apache/netbeans#4643] [Q&A]
hello, i'm trying to make an input that take a greek name and give an answer, the problem is that the answer is like: Hello ???? nice to meet you. 

`System.out.println("Χαίρεται, ποιο είναι το όνομα σου:");
        String theName = input.nextLine();
        myAccount.setName(theName);
        System.out.println();

        System.out.printf("Πολύ ωραίο όνομα" +  myAccount.getName());`

the right result should: Πολύ ωραίο όνομα Χρήστο.
the result now is: Πολύ ωραίο όνομα ?????.

so the problem is in input, i tried in file conf to put utf-8 as it says here https://www.muehlencord.de/wordpress/2015/10/16/netbeans-utf8-encoding/ but no difference, even after reboot.

**Comment by duoduobingbing** [comment]
Launching Netbeans with `-J-Dfile.encoding=UTF-8 -J-Dsun.stdout.encoding=UTF-8 -J-Dstdout.encoding=UTF-8` and your project with `-Dfile.encoding=UTF-8 -Dsun.stdout.encoding=UTF-8 -Dstdout.encoding=UTF-8` should probably fix this issue.

You can verify the project-side by printing out
```
System.out.println(Charset.defaultCharset());
System.out.println(System.getProperty("file.encoding"));
System.out.println(System.getProperty("stdout.encoding"));
System.out.println(System.getProperty("sun.stdout.encoding"));
```
Those should all print `utf-8`

**Comment by Chrisdamia99** [comment]
it didnt work... but thank you for ur try my mate
